docs/guideis/workflow.md

```diff
@@ -0,0 +1,29 @@
+
+
+
+1. Choose or create an [Issue](https://github.com/open-marketplace-applications/dshop/issues).
```

Comment:
Without using any `# Markdown Header` at all, the document does not look complete.